sandbox/examples/event_time_test.cpp

```diff
@@ -26,7 +31,9 @@ int main() {
 
     EventManager em;
     int x = 0;
+    A a;
     em.Subscribe<KeyEvent>(HandleType::SYNC, sub, std::reference_wrapper(x));
+    em.Subscribe<KeyEvent>(HandleType::SYNC, &A::met, &a);
```

Comment:
Add documentation to Subscribe method and describe how to use it with lambdas and methods, because it's hard to tell based on the code.

Reply:
-_-